framework/src/wifi_manager/wifi_manager_state.c

```diff
@@ -322,32 +322,34 @@ wifi_manager_result_e _wifimgr_scan(wifi_manager_scan_config_s *config)
 
 wifi_manager_result_e _handler_on_uninitialized_state(wifimgr_msg_s *msg)
 {
-	wifimgr_evt_e evt = msg->event;
-	if (evt != WIFIMGR_CMD_INIT) {
-		WIFIADD_ERR_RECORD(ERR_WIFIMGR_INVALID_EVENT);
-		return WIFI_MANAGER_FAIL;
-	}
+ wifimgr_evt_e evt = msg->event;
```

Comment:
This looks something wrong. Could you check?

Reply:
Yes, we will revert this as well.